MvvmCross/Droid/Droid/Platform/MvxAndroidSetup.cs

```diff
@@ -158,7 +159,7 @@ protected virtual void RegisterBindingBuilderCallbacks()
             Mvx.CallbackWhenRegistered<IMvxNamespaceListViewTypeResolver>(FillNamespaceListViewTypeResolver);
         }
 
-        protected virtual MvxAndroidBindingBuilder CreateBindingBuilder()
+        protected virtual MvxBindingBuilder CreateBindingBuilder()
```

Comment:
I was already thinking that we should have that, and possible even more base classes. The problem for me is that i don't have too many real world Forms apps where i could test it. We would really appreciate any contributions on this.